### PR TITLE
360 new jumpstart from

### PR DESCRIPTION
**Pull request by smebrahimi**
# Issue Being Addressed

#360 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

- [x] Bug Fix
- [ ] Refactor
- [ ] New Feature
- [ ] Update to Existing Feature
- [ ] Other (state below)

# Description

_For Bug Fixes:

- add showdown package for convert markdown to html

- add style for tags of editor

# How to Test/Reproduce
FAB button

# Additional Context

1. fix preview carousel after multi upload pdf file
2. infinite carousel
3. fix, lable of multiselect when the dropdown is open
4. preview editor
5. prevent invalid validation

### CODE REVIEW

File: src/views/JumpStartAdd.vue [hunk 1]
```diff
@@ -160,37 +160,43 @@
...
               <BaseButton @selectButton="submit" class="w-64" bgType="secondary"> Save </BaseButton>
             </div>
           </template>
         </div>
       </div>
     </template>
-    <template #footer>
-      <NavigationBottom />
-    </template>
   </LayoutFixedFooter>
```
Comment (mort3za) [inline src/views/JumpStartAdd.vue:187]: If there is no NavigationBottom, use LayoutFixedScrollable please.

**Comment by bamboochow** [comment]
# Preview Slider Wobble
There's a weird wobble in the page when you move between slider preview images 
gif

# Remove Autofill
Remove date autofill in Email Subject Line
image

# Incorrect Error Message 
Error messages for tags show "To is required". Tags are not required to send a Jumpstart. 
image

**Comment by mort3za** [comment]
The image (chattanooga) should be responsive. add these classes: `class="h-8 md:h-16 mr-2"` @smebrahimi

File: src/views/JumpStartAdd.vue [hunk 2]
```diff
@@ -129,7 +128,7 @@
                 <template v-slot:noOptions>No options available</template>
               </Multiselect>
               <span v-if="$v.tags.$error" class="text-xs text-error pl-error-message">
-                To is required
+                Tags is required
```
Comment (mort3za) [inline src/views/JumpStartAdd.vue:131]: "At least one tag is required"

**Comment by bamboochow** [comment]
# Tag Requirement 
Tag is NOT required @mort3za @smebrahimi 
I mentioned this in the above comment as well

**Comment by smebrahimi** [comment]
@bamboochow That fixed

**Pull request merged**